src-tauri/Cargo.toml

```diff
@@ -1,12 +1,15 @@
 [package]
 name = "wooting-macros"
-version = "1.0.2"
+version = "1.1.0-pre-release"
+edition = "2021"
 description = "Macro software made by Wooting."
-authors = ["Jeroen N", "Erik S", "Simon W", "David M", "Khang N"]
+authors = ["Jeroen N", "Erik S", "Simon W", "David M <1900179+medzernik@users.noreply.github.com>", "Khang N", "Pola M"]
 license = "GPL-3.0-only"
 repository = "https://github.com/WootingKb/wooting-macros"
-edition = "2021"
-
+readme = "../README.md"
+homepage = "https://wooting.io/wootomation"
+keywords = ["input", "macros", "wooting"]
```

Comment:
It's kinda redundant to add these when it's never going to be published to crates.io?

Reply:
I was thinking maybe it would be useful to publish wootomation eventually, while I was trying to clean up the TOML, this came to mind. Not sure how realistic or useful it would be tho, so I guess i'll remove it

Reply:
You can't really publish a project like this to crates.io I'm pretty sure, generally you submit source code, not a compilation. So highly doubt it's possible unless Tauri is able to integrate more deeply with Cargo

Reply:
Removed in: [9020768](https://github.com/WootingKb/wooting-macros/pull/194/commits/9020768bfb83bc75d8b211030ffc83a356e85231)